app/models/image.rb

```diff
@@ -1,2 +1,3 @@
 class Image < ActiveRecord::Base
+  belongs_to :performer
```

Comment:
this will end up being a polymorphic association - but i don't think that gets there until towards the end of the requirements. this is good for now though!